README.md

```diff
@@ -1,12 +1,12 @@
 
 [![Build Status](https://travis-ci.org/daostack/arc.js.svg?branch=master)](https://travis-ci.org/daostack/arc.js)
-[![NPM Package](https://img.shields.io/npm/v/daostack-arc.js.svg?style=flat-square)](https://www.npmjs.org/package/daostack-arc.js)
+[![NPM Package](https://img.shields.io/npm/v/@daostack/arc.js.svg?style=flat-square)](https://www.npmjs.org/package/@daostack/arc.js)
 
 # Mission Statement
 
-DaoStack-Arc.js is a javascript library providing access to DAOStack Arc ethereum smart contracts (Daostack-Arc).
+DAOstack Arc.js is a javascript library providing access to DAOstack Arc ethereum smart contracts.
 
-Daostack-Arc is a widely open collaboration to build the basic framework for [Decentralized Autonomous Organizations](https://en.wikipedia.org/wiki/Decentralized_autonomous_organization) (DAO) through bootstrap, or dogfooding.  Find the DaoStack-Arc repository [here](https://github.com/daostack/daostack).
+DAOstack Arc is a widely open collaboration to build the basic framework for [Decentralized Autonomous Organizations](https://en.wikipedia.org/wiki/Decentralized_autonomous_organization) (DAO) through bootstrap, or dogfooding.  Find the DAOstack Arc repository [here](https://github.com/daostack/arc).
```

Comment:
> DAOstack Arc is a widely open collaboration to build the basic framework for [Decentralized Autonomous Organizations](https://en.wikipedia.org/wiki/Decentralized_autonomous_organization) (DAO) through bootstrap, or dogfooding. 

This is true for DAOstack not for ARC.

Reply:
Please save documentation comments for the forthcoming documentation PR.  The merges (when rebasing) become increasingly difficult otherwise....thanks!

Reply:
(I realize I made a mistake in changing the documentation upon which you are commenting -- I am going to suffer for it for the same reasons.)

Reply:
Hey, you submitted the changes on this readme.
The readme is the face of this repo.

Reply:
@orenyodfat But if you insist on this change, please let me know ASAP, thanks!

Reply:
This root readme need to be update.
The Arc Configuration ?? section also include wrong info.
I guess that you can submit other pr for the readme though it need to be done

Reply:
@orenyodfat Documentation is being done in a separate PR.  But I have fixed the bit you referred to at the top.